monitor.py

```diff
@@ -244,7 +246,7 @@ class TimelineMonitor(Monitor):
     OVERLAP = datetime.timedelta(minutes=5)
 
     def run_once(self, progress):
-        if progress.finish in (None, progress.NO_VALUE):
+        if progress.finish is None:
```

Comment:
Now that the default value is None, we're never going to start off with `finish` set to `CLEAR_VALUE` -- that's something that only happens on the way _out_ of `run_once`.